From 03a4f314cea1ba46f51462d54b35fb68dfb11421 Mon Sep 17 00:00:00 2001
From: Wayne Zou <b36644@freescale.com>
Date: Fri, 10 Feb 2012 16:42:07 +0800
Subject: ENGR00174243 MIPI_DSI: mipi dsi panel enable should be after IPU
 init

MIPI_DSI: mipi dsi panel enable should be after IPU init,
due to ENGR00173962 change in the mxc_ipuv3_fb.c

Signed-off-by: Wayne Zou <b36644@freescale.com>
---
 drivers/video/mxc/mipi_dsi.c |    1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/video/mxc/mipi_dsi.c b/drivers/video/mxc/mipi_dsi.c
index 244f2d8..fa00ce1 100644
--- a/drivers/video/mxc/mipi_dsi.c
+++ b/drivers/video/mxc/mipi_dsi.c
@@ -884,7 +884,6 @@ static struct mxc_dispdrv_driver mipi_dsi_drv = {
 	.name	= DISPDRV_MIPI,
 	.init	= mipi_dsi_disp_init,
 	.deinit	= mipi_dsi_disp_deinit,
-	.enable = mipi_dsi_power_on,
 	.disable = mipi_dsi_power_off,
 };
 
-- 
1.7.9.5

